content/models.py

```diff
@@ -15,7 +15,7 @@ def __str__(self):
 
 
 class Ourteam(models.Model):
-    img = models.ImageField(upload_to='uploads')
+    img = models.ImageField(blank=True, null=True)
```

Comment:
I thought django does it automatically? on my other project it's directly uploaded to the uploads folder.